backend/src/main/kotlin/blue/mild/covid/vaxx/dto/request/PhoneNumberDtoIn.kt

```diff
@@ -0,0 +1,6 @@
+package blue.mild.covid.vaxx.dto.request
+
+data class PhoneNumberDtoIn(
+    val number: String,
```

Comment:
a to number uz obsahuje i ten country code, nebo ne? protoze v tom logu `phoneNumber=` vypisujes jenom `number`

Reply:
apropo tak sem muzes pridat dokumentaci, co teda v tehdle dvou vecech vlastne je

Reply:
Pridano.